src/kaleidoscope/device/technomancy/Atreus2.h

```diff
@@ -64,10 +64,12 @@ ATMEGA32U4_KEYBOARD(
 }
 }
 
-typedef kaleidoscope::device::technomancy::Atreus2 Device;
+EXPORT_DEVICE(kaleidoscope::device::technomancy::Atreus2)
 
 }
 
+#ifndef KALEIDOSCOPE_VIRTUAL_BUILD
```

Comment:
Since this is just a legacy definition that should never be called anymore, does the simulator break without the ifndef? (Can we drop the guard in cases like this?)

Reply:
By forward declaring the named device types in case of virtual builds, the types can be used and the legacy external references can be defined without `#ifndef` guards. I force-pushed an update.